maro/cli/inspector/cim_dashboard.py

```diff
@@ -0,0 +1,473 @@
+import math
+import os
+
+import altair as alt
+import numpy as np
+import pandas as pd
+import streamlit as st
+
+import maro.cli.inspector.dashboard_helper as helper
+
+from .params import CIMItemOption
+from .params import GlobalFilePaths as Gfiles
+from .params import GlobalScenarios
+from .visualization_choice import CIMIntraViewChoice, PanelViewChoice
+
+
+def start_cim_dashboard(source_path: str, epoch_num: int, prefix: str):
+    """Entrance of cim dashboard.
+
+    Args:
+        source_path (str): Data folder path.
+        epoch_num (int) : Number of data folders.
+        prefix (str): Prefix of data folders.
+    """
+    option = st.sidebar.selectbox(
+        "Data Type",
+        PanelViewChoice._member_names_)
+    if option == PanelViewChoice.Inter_Epoch.name:
+        render_inter_view(source_path, epoch_num)
+    elif option == PanelViewChoice.Intra_Epoch.name:
+        render_intra_view(source_path, epoch_num, prefix)
+    else:
+        pass
+
+
+def render_inter_view(source_path: str, epoch_num: int):
+    """Show CIM summary plot.
+
+    Args:
+        source_path (str): Data folder path.
+        epoch_num (int): Number of snapshot folders.
+    """
+    helper.render_h1_title("CIM Inter Epoch Data")
+    sample_ratio = helper.get_holder_sample_ratio(epoch_num)
+    # get epoch sample num
```

Comment:
comment style: Start with a capital letter, end with a period. Same to other comments.